tests/phpunit/tests/cache.php

```diff
@@ -415,4 +415,67 @@ public function test_wp_cache_delete_multiple() {
 
 		$this->assertSame( $expected, $found );
 	}
+
+	/**
+	 * @ticket 4476
+	 * @ticket 9773
+	 *
+	 * test wp_cache_flush_group
+	 *
+	 * @covers ::wp_cache_flush_group
+	 */
+	public function test_wp_cache_flush_group() {
+		$key = 'my-key';
```

Comment:
@pbearne You either need to implement `flush_group()` in `tests/phpunit/includes/object-cache.php`, or we should skip this test.

```suggestion
		if ( ! method_exists( $this->cache, 'flush_group' ) ) {
			$this->markTestSkipped( 'This test requires a flush_group() method on the cache object.' );
		}

		$key = 'my-key';
```